troubleshooting/debugging.md

```diff
@@ -493,3 +494,29 @@ To work around this issue, we suggest using the following option so that system
 ```
 net.dns.mode LEGACY
 ```
+
+### FAQ
+
+#### AWS Go Plugins vs AWS Core C Plugins
+
+When AWS for Fluent Bit first launched in 2019, we debuted with a set of external output plugins written in [Golang and compiled using CGo](https://docs.fluentbit.io/manual/v/1.2/development/golang_plugins). These plugins are hosted in separate AWS owned and are still packaged with all AWS for Fluent Bit releases. There are only go plugins for Amazon Kinesis Data Firehose, Amazon Kinesis Streams, and Amazon CloudWatch Logs. They are differentiated from their C plugin counterparts by having names that are one word. This is the name that you use when you define an output configuration:
+
+```
+[OUTPUT]
+    Name cloudwatch
```

Comment:
Did you intend to prefix many of these items with Name $thing?

Reply:
Yes because that is what is relavant to the customer and what they ask- how do I know based on looking at a config whether it is a go plugin or a C plugin